barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/client_ivc_recursive_verifier.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "barretenberg/client_ivc/client_ivc.hpp"
+#include "barretenberg/stdlib/honk_recursion/verifier/decider_recursive_verifier.hpp"
+
+namespace bb::stdlib::recursion::honk {
+class ClientIvcRecursiveVerifier_ {
```

Comment:
I wonder whether we should template this by builder.. unlikely we're going to use MegaCircuitBuilder but it's worth having the mechanism to quickly switch

Reply:
I don't think this is worth doing, at least not until we need it. Using mega requires using Goblin so it changes the whole dynamic of what this class does